docs/_articles/en/apex/apex-get-started.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Quick Start Apex Development
+lang: en
+---
+  
+## Overview
+Complete this Quick Start to create, test and debug a workflow that is created using an Apex class, and sets off a Apex trigger.
+
+We’ll create a custom object called `Book`, and add a custom field called `Price` to that custom object. We’ll update and test the updated Price field using Apex and learn how to work with Apex Code using the Salesforce Extensions.
+
+
+Before you start, install [Salesforce Extensions for Desktop](./en/../../vscode-desktop/install.md) or [Set Up Code Builder](./en/../../codebuilder/cb-setup.md)
+
+
+### Create a Custom Object
+1. Log in to your Sandbox or Developer org.
+2. Go to **Setup** > **Object Manager** > **Create** > **Custom Object**.
+3. Enter `Book` for the label.
+4. Enter `Books` for the plural label.
+5. Click **Save.**
+6. In the **Custom Fields & Relationships** section of the Book detail page, click **New**.
+7. Select `Number` for the data type and click **Next**.
+8. Enter `Price` for the field label.
+9. Enter `16` in the length text box.
+10. Enter `2` in the decimal places text box, and click Next.
+11. Click **Next** to accept the default values for field-level security.
+12. Click **Save**.
+
+### Add an Apex Class
+1. In VS Code, connect to the org that you added the custom object to using [Org Picker or Command Palette](../user-guide/default-org.md).
+2. Run the `SFDX: Refresh SObject Definitions` command from the Command Palette to get code completion suggestions for your SObjects related code.
+3. Click the Cloud icon in the Activity Bar to open the Org Browser.
+4. Scroll down to Custom Object and locate the `Book_c` object. Click the retrieve icon to run `SFDX: Retrieve Source from Org`.
+5. From the Command Palette run `SFDX:Create Apex Class` and create a class called `MyBooks`. 
+6. Add a method called `applyDiscount` to this class. 
+   
+We’ll make this method both public and static. Because it’s a static method, you don't need to create an instance of the class to access the method—you can just use the name of the class followed by a dot (.) and the name of the method.
+
+This method takes one parameter, a list of Book records, which is assigned to the variable books. The method iterates through a list of books and applies a 10% discount to the current book price:
+
+```
+public with sharing class MyBooks {
+    public MyBooks() {
+
+    }
+    public static void applyDiscount(Book__c[] books) {
+        for (Book__c b : books){
+           b.Price__c *= 0.9;
+        }
+     }
+}
+```
+Next we’ll add a trigger that calls this `applyDiscount` method. 
+
+### Add an Apex Trigger
+An Apex trigger is a piece of code that executes before or after records of a particular type are inserted, updated, or deleted from the Lightning Platform database. Every trigger runs with a set of context variables that provide access to the records that caused the trigger to fire.
```

Comment:
Is "Lightning Platform" database the official name? I don't know, just asking.

Reply:
I see inconsistent use all over. Changing it to be more generic. Thanks!